### ISSUE TITLE
Random fail while syncing with ipc on

### ISSUE THREAD

**Issue by NikVolf** [ethcore/parity#1903]
```
       0+    0 Qed    #521459    2/ 2/25 peers     14 MiB db    3 MiB chain  236 KiB queue   48 KiB sync
thread '<unnamed>' panicked at 'ipc deserialization error, aborting: Serialization(BinaryConvertError { member_tree: ["transactions"], kind: InconsistentBoundaries })', ../src/libcore/result.rs:785
note: Run with `RUST_BACKTRACE=1` for a backtrace.
thread 'IO Worker #0' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Custom(Custom { kind: TimedOut, error: StringError("Connection timed out") }) })', ../src/libcore/result.rs:785
note: Run with `RUST_BACKTRACE=1` for a backtrace.
thread 'IO Worker #2' panicked at 'called `Result::unwrap()` on an `Err` value: "PoisonError { inner: .. }"', ../src/libcore/result.rs:785
thread 'IO Worker #3' panicked at 'called `Result::unwrap()` on an `Err` value: "PoisonError { inner: .. }"', ../src/libcore/result.rs:785
thread 'IO Worker #1' panicked at 'called `Result::unwrap()` on an `Err` value: "PoisonError { inner: .. }"', ../src/libcore/result.rs:785
```

**Comment by arkpar** [comment]
Still happens on 53b22da

**Comment by NikVolf** [comment]
@arkpar 
Exactly this?
`BinaryConvertError { member_tree: ["transactions"], kind: InconsistentBoundaries }` ?

```
thread '<unnamed>' panicked at 'ipc deserialization error, aborting: Serialization(BinaryConvertError { member_tree: ["transactions"], kind: InconsistentBoundaries })', ../src/libcore/result.rs:785
```

**Comment by arkpar** [comment]
Don't have the log anymore. It was definitely `Serialization(BinaryConvertError`

**Comment by 5chdn** [comment]
Was this fixed? It's tagged with 1.3 milestone 6 months ago.

**Comment by NikVolf** [comment]
mostly not relevant now
